build.gradle

```diff
@@ -539,13 +536,14 @@ project(':iceberg-delta-lake') {
     implementation project(':iceberg-common')
     implementation project(':iceberg-core')
     implementation project(':iceberg-parquet')
+    implementation platform(libs.jackson.bom)
     implementation "com.fasterxml.jackson.core:jackson-databind"
-    annotationProcessor "org.immutables:value"
-    compileOnly "org.immutables:value"
+    annotationProcessor libs.immutables.value
+    compileOnly libs.immutables.value
 
-    compileOnly "io.delta:delta-standalone_${scalaVersion}"
+    compileOnly "io.delta:delta-standalone_${scalaVersion}:${libs.versions.deltaStandalone.get()}"
```

Comment:
is there a better way to define this? Also why does this need a version now?

Reply:
Because it's a  dynamic dependency that include scalaVersion, it seem that there is no way to express it purely using versionCaralog using nice looking path like libs.delta.standalone